app/mailers/heya/campaign_mailer.rb

```diff
@@ -1,6 +1,7 @@
 module Heya
   class CampaignMailer < ApplicationMailer
-    layout "heya/campaign_mailer"
+    DEFAULT_LAYOUT = "heya/campaign_mailer".freeze
```

Comment:
I don't really care if we freeze the string here. I don't think I am in other places (and I prefer to use `frozen_string_literal`).

```suggestion
    DEFAULT_LAYOUT = "heya/campaign_mailer"
```